src/app/main.rs

```diff
@@ -476,47 +492,55 @@ where
                 ..oc::Node::default()
             };
             let span_exporter = SpanExporter::new(trace_collector, node, spans_rx, span_metrics);
-            task::spawn(span_exporter.map_err(|e| {
-                error!("span exporter failed: {}", e);
-            }));
+            tokio::spawn(
+                span_exporter
+                    .map_err(|e| {
+                        error!("span exporter failed: {}", e);
```

Comment:
Nit/TIOLI: consider a structured field for the error:
```suggestion
                        error!(error = %e, "span exporter failed");
```